### PR TITLE
docs: add section for developer to add new keys

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

File: docs/i18n.md [hunk 1]
```diff
@@ -45,3 +45,13 @@ officially added.
...
 Or use a build of the app with an inline editor (see above).
+
+## For developers
+
+New keys can be added to `frontends/web/src/locales/en/app.json`
+
+Keys have to be in alphabetical order, this can be automatically fixed by running
+
+```bash
+npx locize-cli format frontends/web/src/locales --format json
```
Comment (x1ddos) [inline docs/i18n.md:56]: Can we add this to the scripts/ci.sh? There's already a relevant section in there:
https://github.com/digitalbitbox/bitbox-wallet-app/blob/144eea22c8cf74100fc277ce1ea39d4f89cc0096/scripts/ci.sh#L27-L32

Comment (x1ddos) [reply docs/i18n.md:56]: Should this simply be `make locize-fix`?

Comment (x1ddos) [reply docs/i18n.md:56]: and maybe add it to the "Repo maintainers" section above?

Comment (thisconnect) [reply docs/i18n.md:56]: added "developers" for other contributors that are not maintainers and might run into this issue in the future. do you prefer if it is under "Repo maintainers"?

Comment (x1ddos) [reply docs/i18n.md:56]: sounds good

Comment (x1ddos) [reply docs/i18n.md:56]: I mean sounds good as is.

Comment (thisconnect) [reply docs/i18n.md]: replaced with `make locize-fix` and fixed squashed pushed

File: docs/i18n.md [hunk 2]
```diff
@@ -45,3 +45,13 @@ officially added.
 Login to locize.io to the bitbox-wallet-app project.
 
 Or use a build of the app with an inline editor (see above).
+
+## For developers
+
+New keys can be added to `frontends/web/src/locales/en/app.json`
```
Comment (x1ddos) [inline docs/i18n.md:51]: Dot `.` and the end of the sentence, please.

**Pull request merged**